x/oracle/simulations/decoder.go

```diff
@@ -0,0 +1,47 @@
+package simulation
+
+import (
+	"bytes"
+	"fmt"
+
+	gogotypes "github.com/gogo/protobuf/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/kv"
+
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// NewDecodeStore returns a decoder function closure that unmarshals the KVPair's
+// Value to the corresponding oracle type.
+func NewDecodeStore(cdc codec.Codec) func(kvA, kvB kv.Pair) string {
+	return func(kvA, kvB kv.Pair) string {
+		switch {
+		case bytes.Equal(kvA.Key[:1], types.KeyPrefixExchangeRate):
```

Comment:
nit: I personally prefer putting a newline after each `case` body. It makes it slightly easier to read IMO.